upload/modules/Store/gateways/PayPalBusiness/gateway.php

```diff
@@ -253,17 +324,23 @@ public function handleListener(): void {
 
                             break;
                         case 'BILLING.SUBSCRIPTION.CREATED':
-                            $id = $response->resource->id;
+                            // Subscription created
+                            $subscription = new Subscription($response->resource->id, 'agreement_id');
+                            if ($subscription->exists()) {
+                                $subscription->update([
+                                    'status_id' => 1
```

Comment:
I feel like using an enum would be beneficial here to make it clearer what exactly the status is. Using a number is very vague and hard to remember